File: java/mix/D03CsvAggregate.java
import java.util.HashMap;
import java.util.Map;

public class D03CsvAggregate {
    public static Map<String, Double> aggregateLines(Iterable<String> lines) {
        Map<String, Double> acc = new HashMap<>();
        for (String line : lines) {
            String[] parts = line.strip().split(",");
            String key = parts[0];
            double val = Double.parseDouble(parts[1]);
            acc.put(key, acc.getOrDefault(key, 0.0) + val);
        }
        return acc;
    }
}

File: javascript/mix/D03CsvAggregate.js
function aggregateLines(lines) {
    const acc = {};
    for (const line of lines) {
        const parts = line.trim().split(",");
        const key = parts[0];
        const val = parseFloat(parts[1]);
        acc[key] = (acc[key] ?? 0) + val;
    }
    return acc;
}

module.exports = { aggregateLines };
